monai_ex/transforms/utility/array.py

```diff
@@ -9,6 +9,7 @@
 from monai.transforms import DataStats, SaveImage, CastToType
 from monai.config import NdarrayTensor, DtypeLike
 from monai_ex.utils import convert_data_type_ex
+from skimage.morphology import skeletonize_3d
```

Comment:
Please use `optional_import`. Refer to [monai example](https://github.com/Project-MONAI/MONAI/blob/0c2fbac6eb55a546886921d9add66be9688d9775/monai/transforms/utils.py#L62), and raise error if skimage is not installed.